goal_src/jak2/engine/ai/enemy-h.gc

```diff
@@ -5,5 +5,759 @@
+    (enemy-method-77 () none 77)
+    (enemy-method-78 () none 78)
+    (enemy-method-79 () none 79)
+    (enemy-method-80 () none 80)
+    (enemy-method-81 () none 81)
+    (enemy-method-82 () none 82)
+    (enemy-method-83 () none 83)
+    (enemy-method-84 () none 84)
+    (enemy-method-85 () none 85)
+    (enemy-method-86 () none 86)
+    (enemy-method-87 () none 87)
+    (enemy-method-88 () none 88)
+    (enemy-method-89 () none 89)
+    (enemy-method-90 () none 90)
+    (enemy-method-91 () none 91)
+    (enemy-method-92 () none 92)
+    (enemy-method-93 () none 93)
+    (enemy-method-94 () none 94)
+    (enemy-method-95 () none 95)
+    (enemy-method-96 () none 96)
+    (enemy-method-97 () none 97)
+    (enemy-method-98 () none 98)
+    (enemy-method-99 () none 99)
+    (enemy-method-100 () none 100)
+    (enemy-method-101 () none 101)
+    (enemy-method-102 () none 102)
+    (enemy-method-103 () none 103)
+    (enemy-method-104 () none 104)
+    (enemy-method-105 () none 105)
+    (enemy-method-106 () none 106)
+    (enemy-method-107 () none 107)
+    (enemy-method-108 () none 108)
+    (enemy-method-109 () none 109)
+    (enemy-method-110 () none 110)
+    (enemy-method-111 () none 111)
+    (enemy-method-112 () none 112)
+    (enemy-method-113 () none 113)
+    (enemy-method-114 () none 114)
+    (enemy-method-115 () none 115)
+    (enemy-method-116 () none 116)
+    (enemy-method-117 () none 117)
+    (enemy-method-118 () none 118)
+    (enemy-method-119 () none 119)
+    (enemy-method-120 () none 120)
+    (enemy-method-121 () none 121)
+    (enemy-method-122 () none 122)
+    (enemy-method-123 () none 123)
+    (enemy-method-124 () none 124)
+    (enemy-method-125 () none 125)
+    (enemy-method-126 () none 126)
+    (enemy-method-127 () none 127)
+    (enemy-method-128 () none 128)
+    (enemy-method-129 () none 129)
+    (enemy-method-130 () none 130)
+    (enemy-method-131 () none 131)
+    (enemy-method-132 () none 132)
+    (enemy-method-133 () none 133)
+    (enemy-method-134 () none 134)
+    (enemy-method-135 () none 135)
+    (enemy-method-136 () none 136)
+    )
+  )
+
+;; definition for method 3 of type enemy
+(defmethod inspect enemy ((obj enemy))
+  (when (not obj)
+    (set! obj obj)
+    (goto cfg-79)
+    )
+  (let ((t9-0 (method-of-type process-focusable inspect)))
+    (t9-0 obj)
+    )
+  (format #t "~2Tenemy-flags: #x~X : (enemy-flag " (-> obj enemy-flags))
+  (let ((s5-0 (-> obj enemy-flags)))
+    (if (= (logand (enemy-flag actor-pause-backup) s5-0) (enemy-flag actor-pause-backup))
+        (format #t "lock-focus ")
+        )
+    (if (= (logand (enemy-flag recover) s5-0) (shl 8 32))
+        (format #t "death-start ")
+        )
+    (if (= (logand s5-0 (enemy-flag jump-check-blocked)) (enemy-flag jump-check-blocked))
+        (format #t "enable-on-active ")
+        )
+    (if (= (logand (enemy-flag directed-ready) s5-0) (enemy-flag directed-ready))
+        (format #t "checking-water ")
```

Comment:
hm, it looks like these flag values aren't quite right - I'd expect to see the print statements match the flag they are testing (so this one should be `directed-ready`)